index.js

```diff
@@ -119,7 +120,7 @@ function buildMemoizer (options) {
                     })
                   }, optExt.maxAge)
                 }
-                fs.writeFile(filePath, resultString, cb)
+                steno.writeFile(filePath, resultString, cb)
```

Comment:
We can use `write-file-atomic` by npm: https://github.com/npm/write-file-atomic

Reply:
Feel free to take over this PR. :) I'm not using with this project anymore, so I won't get around to test it.